templates/singleton/boil_queries.tpl

```diff
@@ -82,7 +82,12 @@ func deleteOneToOneConflictsBeforeMerge(tx boil.Executor, conflict conflictingUn
 
 func deleteOneToManyConflictsBeforeMerge(tx boil.Executor, conflict conflictingUniqueKey, primaryID uint64, secondaryID uint64) error {
 	conflictingColumns := strmangle.SetComplement(conflict.columns, []string{conflict.objectIdColumn})
-
+    var objectIDIndex int
+    for i, column := range conflict.columns {
+        if column == conflict.objectIdColumn {
+            objectIDIndex = i
```

Comment:
you can `break` after this

Reply:
woops..I thought I did that...like literally was thinking it and never added it lol.